test/box/fix-memory-corruption.test.lua

```diff
@@ -0,0 +1,40 @@
+env = require('test_run')
+fiber = require('fiber')
+net_box = require('net.box')
+
+test_run = env.new()
+
+test_run:cmd('create server test_server with script="box/fix-memory-corruption-server.lua"')
+test_run:cmd("setopt delimiter ';'")
+for i = 1, 50 do
+    test_run:cmd(string.format('create server test_client_%d with script=\"box/fix-memory-corruption/fix-memory-corruption-client_%d.lua\"', i, i))
+end;
+test_run:cmd("setopt delimiter ''");
+
+test_run:cmd('start server test_server')
+test_run:cmd("setopt delimiter ';'")
+for i = 1, 50 do
+    test_run:cmd(string.format('start server test_client_%d with wait=False, wait_load=False', i))
```

Comment:
You do realize that having 100 processes for such a simple test is a total overkill? We run tests sometimes tens time per day, the tests must be quick and easy to run in a console when possible. So as we could reproduce and debug them out of test-run.

Reply:
Oh god, I just noticed that you also added 50! files. Please, lets be sane.

Reply:
I delete all this files)) We need at least 20 processes to get segmentation fault on master branch quite often. The test completes in about 10 seconds.